corebehrt/functional/creators.py

```diff
@@ -9,26 +9,26 @@
                               get_time_difference, normalize_segments_df)
 
 
-def create_ages(concepts: pd.DataFrame, birthdates: dict) -> pd.DataFrame:
+def create_ages(concepts: pd.DataFrame, patients_info: pd.DataFrame) -> pd.DataFrame:
     """Creates the AGE column"""
-    concepts['AGE'] = get_time_difference(concepts['TIMESTAMP'], concepts['PID'].map(birthdates))
+    birthdates = patients_info.set_index('PID')['BIRTHDATE']
+    concepts['age'] = get_time_difference(concepts['TIMESTAMP'], concepts['PID'].map(birthdates))
     return concepts
 
 def create_abspos(concepts: pd.DataFrame, origin_point: datetime) -> pd.DataFrame:
     """Creates the ABSPOS column"""
-    concepts['ABSPOS'] = get_abspos_from_origin_point(concepts['TIMESTAMP'], origin_point)
+    concepts['abspos'] = get_abspos_from_origin_point(concepts['TIMESTAMP'], origin_point)
     return concepts
 
 def create_segments(concepts: pd.DataFrame, segment_col='ADMISSION_ID') -> pd.DataFrame:
     """ Creates the SEGMENT column (the normalize segments_df can do this) """
-    concepts['SEGMENT'] = normalize_segments_df(concepts, segment_col)
-    return concepts
+    return normalize_segments_df(concepts, segment_col)
```

Comment:
This doesn't follow the format as the other create functions, where we create the new column of concepts inside the function - However, I think the other functions should follow this format